testsuite/src/test/java/org/jboss/intersmash/testsuite/provision/openshift/WildflyOperatorProvisionerTest.java

```diff
@@ -48,7 +45,7 @@ private static WildflyOperatorProvisioner initializeOperatorProvisioner() {
 					@Override
 					public WildFlyServer getWildflyServer() {
 						return new WildFlyServerBuilder(getName())
-								.applicationImage(IntersmashConfig.wildflyImageURL())
+								.applicationImage("quay.io/wildfly/wildfly")
```

Comment:
Is there some reason for hardcoding wf image here?

Reply:
Hi @marekkopecky - yes, the currently existing properties exposed by IntersmashConfig, would not relate to a "runnable" WildFly server, but just to the s2i (builder and runtime) images.
Those aren't good for what we want to do here, i.e. test that the WildFLy operator can be run a WildFly application image, even though the server which is execute does not include any deployment, which is expected.

At the moment we don't need another property, though, but we might want to introduce such property later on, after evaluating the behnavior with - let's say - a productized version of the operator. Currently we're trying to enable all the provisioners for just the community versions, in order to set a baseline.

Reply:
Understand, ok, resolving this thread. Thanks!